### PR TITLE
Fe feature create userpf

### PR DESCRIPTION
**Pull request by leah-is-offline**
profile html and css
checks is user can change displayname, username, github.

### CODE REVIEW

File: citrus_network/citrus_home/views.py [hunk 1]
```diff
@@ -213,6 +231,67 @@ def manage_profile(request, id):
...
+            #raise forms.ValidationError(u'Display Name "%s" is already in use.' % new_displayName)
+        else:
+            print("you did not change your display name - this one is already yours")
+            return True
+    else:
+        print("display name is available - no one had it yet")
+        return True
+
+def validate_github(profile,  new_github):
+    print("HERE IS USERS DISPLAY NAME")
```
Comment (nhtnhan) [inline citrus_network/citrus_home/views.py:281]: shouldn't this be github uri?

**Review by nhtnhan — COMMENTED** [review]
just commented on a minor print statement

File: citrus_network/citrus_home/views.py (hunk 1 above)
Comment (leah-is-offline) [reply citrus_network/citrus_home/views.py:281]: yes

**Review by nhtnhan — APPROVED** [review]
looks good!

**Pull request merged**